src/manual/design/apis/hawk/signed-urls.md

```diff
@@ -1,15 +1,16 @@
 ---
+filename: design/apis/hawk/signed-urls.md
 title: Pre-signed URLs
 order: 27
 ---
 
 Hawk allows you to generate a _bewit_ signature for any `GET` request. Including
 this _bewit_ signature in your request will then authenticate and authorize the
-request. All TaskCluster APIs support authentication of `GET` requests using
+request. All Taskcluster APIs support authentication of `GET` requests using
 these bewit signatures. And you'll find that the official
 [taskcluster-client](https://github.com/taskcluster/taskcluster-client)
 offers an API for generating these signatures.
 
 Pre-signed URLs are most often used to provide signed URLs allowing another
-application to download a private artifact without including TaskCluster
-authentication headers.
+application to download a private artifact without including Taskcluster
+authentication headers.
```

Comment:
Maybe unsigned, but the same style of URL is also used for non-private artifacts.  Maybe we can rephrase it to say that it's mainly used when you'd like to use your own HTTP client but don't want to have to figure out the right values for auth, and use the example of a private artifact download?